src/cultural_algorithm.py

```diff
@@ -0,0 +1,140 @@
+import math
+import random
+import numpy as np
+
+POPULATION_SIZE = 30
+GENERATIONS = 100
+ACCEPT_NUMBER = 2
+
+
+def value_fitness(individual):
+    if len(individual) != 2:
+        raise ValueError(f"The length of individual is incorrect!")
```

Comment:
Say what the length _was_ such that it threw the exception. Something like this. 
`f"The length of individual is incorrect: {len(individual)}"